tools/buildsys/src/builder.rs

```diff
@@ -89,6 +95,19 @@ impl VariantBuilder {
         let version_image = getenv("BUILDSYS_VERSION_IMAGE")?;
         let version_build = getenv("BUILDSYS_VERSION_BUILD")?;
         let output = getenv("BUILDSYS_OUTPUT_DIR")?;
+        let image_format = match image_format {
+            Some(ImageFormat::IMG) | None => String::from("img"),
+            Some(ImageFormat::VMDK) => String::from("vmdk"),
+        };
+        let image_options = match image_options {
+            Some(options_string) => options_string.join(","),
```

Comment:
nit: `options_string` isn't a string here, let's avoid suggesting it is:
```suggestion
            Some(opts) => opts.join(","),
```

Reply:
Makes sense to me, but it looks like we're just going to remove image_options for now.